src/qt/walletframe.cpp

```diff
@@ -103,7 +103,8 @@ void WalletFrame::setCurrentWallet(WalletModel* wallet_model)
     walletView->updateGeometry();
 
     walletStack->setCurrentWidget(walletView);
-    walletView->updateEncryptionStatus();
```

Comment:
In commit "qt: Add WalletFrame::currentWalletSet signal" (c0f84d9d4354e128b16524c14b6c2dadb891a1b6)

Note: At was unclear to me at first if this commit was changing behavior. But it does not seem to change anything since the previous `updateEncryptionStatus` just called `updateWalletStatus` indirectly.